package.json

```diff
@@ -24,7 +24,8 @@
     "terser-output": "1.0.3",
     "typedoc": "0.22.3",
     "typescript": "4.4.3",
-    "v8-compile-cache": "2.3.0"
+    "v8-compile-cache": "2.3.0",
+    "zlib": "^1.0.5"
```

Comment:
uninstall this , https://www.npmjs.com/package/zlib it's a package from over a decade now nodejs have zlib built inside the runtime ( you can use it by `require("zlib")` ) https://nodejs.org/dist/latest-v16.x/docs/api/zlib.html#zlib_threadpool_usage_and_performance_considerations